Gallery/TaylorDiagrams/NCL_taylor_6.py

```diff
@@ -0,0 +1,76 @@
+"""
+NCL_taylor_6.py
+===============
+This script illustrates the following concepts:
+   - Using geocat-viz Taylor diagram function to create a Taylor diagram. Documentation here: https://geocat-viz.readthedocs.io/en/latest/user_api/generated/geocat.viz.TaylorDiagram.html#geocat.viz.TaylorDiagram
```

Comment:
```suggestion
   - Using geocat-viz `Taylor diagram function <https://geocat-viz.readthedocs.io/en/latest/user_api/generated/geocat.viz.TaylorDiagram.html#geocat.viz.TaylorDiagram>`_ to create a Taylor diagram.
```
This will make it so the words "Taylor diagram function" are a hyperlink to the documentation. You can add links to any doc string this way. We don't do this with the NCL script and plot links since they aren't long and are part of a list and not a sentence.
```
`text <url>`_
```